src/ResourceManager/Consumption/Commands.Consumption/Models/PSUsageDetail.cs

```diff
@@ -12,35 +12,44 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.Consumption.Common;
 using System;
 using System.Collections.Generic;
+using Microsoft.Azure.Commands.Consumption.Common;
 using ApiUsageDetail = Microsoft.Azure.Management.Consumption.Models.UsageDetail;
 using MeterDetails = Microsoft.Azure.Management.Consumption.Models.MeterDetails;
 
 namespace Microsoft.Azure.Commands.Consumption.Models
 {
     public class PSUsageDetail
     {
-        public string Id { get; private set; }
-        public string Name { get; private set; }
-        public string Type { get; private set; }
-        public IDictionary<string, string> Tags { get; private set; }
-        public DateTime? UsageStart { get; private set; }
-        public DateTime? UsageEnd { get; private set; }
+        public string AccountName { get; set; }
+        public string AdditionalProperties { get; set; }
+        public decimal? BillableQuantity { get; set; }
+        public string BillingPeriodId { get; set; }
         public string BillingPeriodName { get; set; }
-        public string InvoiceName { get; set; }
-        public string InstanceName { get; set; }
+        public string ConsumedService { get; set; }
+        public string CostCenter { get; set; }
+        public string Currency { get; set; }
+        public string DepartmentName { get; set; }
+        public string Id { get; private set; }
         public string InstanceId { get; set; }
         public string InstanceLocation { get; set; }
-        public string Currency { get; set; }
-        public decimal? UsageQuantity { get; set; }
-        public decimal? BillableQuantity { get; set; }
-        public decimal? PretaxCost { get; set; }
+        public string InstanceName { get; set; }
+        public string InvoiceId { get; set; }
+        public string InvoiceName { get; set; }
         public bool? IsEstimated { get; set; }
-        public string MeterId { get; set; }
         public MeterDetails MeterDetails { get; set; }
-        public IDictionary<string, string> AdditionalProperties { get; set; }
+        public string MeterId { get; set; }
+        public string Name { get; private set; }
+        public decimal? PretaxCost { get; set; }
+        public string Product { get; set; }
+        public string SubscriptionGuid { get; set; }
+        public string SubscriptionName { get; set; }
+        public IDictionary<string, string> Tag { get; private set; }
```

Comment:
Keep this one as "Tags" because otherwise it is a breaking change.

Reply:
@maddieclayton Is this a misunderstanding? As Tag is a new parameter, it doesn't introduce any breaking change. I will keep its name as `Tag` throughout the pull request.

Reply:
Will keep it as Tags